docs/guides/hub_guide.md

```diff
@@ -1,15 +1,15 @@
-# IOTA Hub migration guide
+# IOTA Hub Migration Guide
 
-> This guide will explain how you can switch from IOTA Hub to using wallet.rs or its bindings instead.
+> This guide outlined how to switch from IOTA Hub to using wallet.rs or its bindings instead.
```

Comment:
Unless this guide no longer outlines the how to switch, I'd suggest the following edit.

```suggestion
This guide outlines how to switch from IOTA Hub to using [Wallet.rs](https://wallet-lib.docs.iota.org/) or its [bindings](https://wallet-lib.docs.iota.org/libraries/index.html) instead.
```

Reply:
Yep!